### ISSUE TITLE
HELLO

### ISSUE THREAD

**Issue by extremlymtorrents** [nk932714/1337x-scrapper#1]
can u meke it to import torrents to database?

**Comment by nk932714** [comment]
importing torrents can be done by scraping each torrent link. Since it didn't provide the magnet link info in single click. 
Scraping each link will increase the load on website. Which might lead on shutdown of this beautiful website. 

But i can do one thing that if you manually click on any link. That link will be loaded to database.(I'll try this in free time.)

**Comment by extremlymtorrents** [comment]
YES ... You can make it to IMPORT only the SEARCH results ! Theat will be Greate 

Your code with my changes https://extremlymtorrents.ws/xsearch.php?word=extremlymtorrents&submit=Submit
